### PR TITLE
feat: export error handling function

### PR DESCRIPTION
**Pull request by makinwab**
Export the [error-handler library](https://github.com/contentful/contentful-management.js/blob/master/lib/error-handler.ts) in contentful-management.js from the sdk-core.

### CODE REVIEW

File: src/error-handler.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * for more details on the data received on the errorResponse.data property
+ * and the expected error codes.
+ * @private
+ */
+export default function errorHandler(errorResponse: AxiosError): never {
+  const { config, response } = errorResponse
+  let errorName
+
+  // Obscure the Management token
+  if (config && config.headers && config.headers['Authorization']) {
```
Comment (marcolink) [inline src/error-handler.ts:17]: Should we also include the check for older node versions?
https://github.com/contentful/contentful.js/blob/5ae9e093f8541f3e6bad8c357ef88229adb85ab0/lib/contentful.js#L106-L108

Comment (makinwab) [reply src/error-handler.ts:17]: We don't need to here. What's deprecated is [_headers](https://nodejs.org/api/deprecations.html#DEP0066)

Comment (marcolink) [reply src/error-handler.ts:17]: That's what I meant. should we also try to obfuscate the token for older node versions (before depreciation)?

Comment (makinwab) [reply src/error-handler.ts:17]: The token is going to be obscured for older node versions here too. The difference between this and what we had in contentful.js is in addition to this config header, we had other places in the response with tokens and so we had to for each of those places obscure the token.
Now that we don't send the entire Axios error, we just need to do this once in this config header.

Comment (marcolink) [reply src/error-handler.ts:17]: Ah right, understood!

File: package.json [hunk 2]
```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "fast-copy": "^2.1.0",
+    "lodash": "^4.17.21",
```
Comment (marcolink) [inline package.json:47]: Did you try [`lodash-es`](https://preview.npmjs.com/package/lodash-es) for better tree shaking support?

**Review by marcolink — APPROVED** [review]

File: test/unit/error-handler-test.spec.ts [hunk 3]
```diff
@@ -0,0 +1,115 @@
+import errorHandler from '../../src/error-handler'
+import { errorMock } from './mocks'
+import { expect } from 'chai'
+import cloneDeep from 'lodash/cloneDeep'
```
Comment (marcolink) [inline test/unit/error-handler-test.spec.ts:4]: can we also switch to `lodash-es` here?

**Pull request merged**